style.css

```diff
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700;900&display=swap');
+
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  background-color: #fff;
+  width: 100%;
+  height: 90px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.mh-logo {
+  display: flex;
+  color: #5969fa;
+  line-height: 20px;
+  text-align: center;
+  margin-left: 24px;
+}
+
+header > .mh-logo {
+  font-size: 18px;
+}
+
+.main-nav {
+  margin-right: 34px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  font-family: Poppins, sans-serif;
+  background-image: url(./Images/bg.png);
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  padding: 90px 24px;
+}
+
+.intro {
+  margin-left: 24px;
+  margin-right: 24px;
+  font-family: Poppins, sans-serif;
+  font-weight: bold;
+  width: 327px;
+  font-size: 52px;
+  line-height: 52px;
+}
+
+.intro-connect {
+  width: 60%;
+  padding-left: 15px;
+  text-decoration: none;
+}
+
+.let-connet {
+  text-decoration: none;
+  color: #7f8cff;
+}
+
+.primary-text {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 60px;
+  color: #091e42;
+  padding-top: 26px;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.37px;
+}
+
+.secondary-text {
+  margin-left: 24px;
+  font-family: Poppins, sans-serif;
+  margin-right: 24px;
+  padding-top: 10px;
+  font-weight: normal;
+  padding-right: 3px;
+  font-style: normal;
+  text-align: justify;
+  text-justify: auto;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  width: 920px;
+  height: 98px;
+  font-size: 20px;
+  line-height: 28px;
+}
```

Comment:
- [ ] You should remove the fixed `width` as it's causing the text to flow out of the window, you should avoid using fixed widths, especially since you're using flexbox to align your items, this should give more room for flexibility, so please fix that. 😁 
![image](https://user-images.githubusercontent.com/20567503/177660422-66005289-10f5-42db-86a1-5c8163c2ae39.png)